packages/math-extras/src/pointExtras.ts

```diff
@@ -0,0 +1,143 @@
+import { Point, IPointData, ObservablePoint } from '@pixi/math';
+
+const mixins: any = {
+    /**
+     * Adds `other` to `this` point and outputs into `outPoint` or a new Point
+     * @method add
+     * @memberof PIXI.Point#
+     * @alias PIXI.ObservablePoint.add
```

Comment:
@ShukantPal is there a way to have this jsdoc be parsed for both `Point` and `ObservablePoint` ?
`@alias` didn't work but maybe there is some other secret word to use?

Reply:
Right now, no :)

Reply:
webdoc can't infer if the prototype assignment is a method or property. It doesn't infer `mixin.add`'s documentation either. I think an @alias feature request would be appropriate.